src/test/mediapipe/inputsidepacketusertestcalc.cc

```diff
@@ -0,0 +1,114 @@
+        RET_CHECK(!cc->Inputs().GetTags().empty());
+        RET_CHECK(!cc->Outputs().GetTags().empty());
+        cc->Inputs().Tag(IN_FP32_TAG).Set<ov::Tensor>();
+        cc->Outputs().Tag(INT32_TAG).Set<ov::Tensor>();
+        cc->Outputs().Tag(INT64_TAG).Set<ov::Tensor>();
+        cc->Outputs().Tag(BOOL_TAG).Set<ov::Tensor>();
+        cc->InputSidePackets().Tag(ISP_STRING.c_str()).Set<std::string>();
+        cc->InputSidePackets().Tag(ISP_INT64.c_str()).Set<int64_t>();
+        cc->InputSidePackets().Tag(ISP_BOOL.c_str()).Set<bool>();
+        MLOG("InputSidePacketUserTestCalc GetContract end");
+        return absl::OkStatus();
+    }
+
+    absl::Status Close(CalculatorContext* cc) final {
+        MLOG("InputSidePacketUserTestCalc Close");
+        return absl::OkStatus();
+    }
+    absl::Status Open(CalculatorContext* cc) final {
+        MLOG("InputSidePacketUserTestCalc Open start");
+        stringParam = cc->InputSidePackets()
+                          .Tag(ISP_STRING.c_str())
+                          .Get<std::string>();
+        boolParam = cc->InputSidePackets()
+                        .Tag(ISP_BOOL.c_str())
+                        .Get<bool>();
+        int64Param = cc->InputSidePackets()
+                         .Tag(ISP_INT64.c_str())
+                         .Get<int64_t>();
+        for (CollectionItemId id = cc->Inputs().BeginId();
+             id < cc->Inputs().EndId(); ++id) {
+            if (!cc->Inputs().Get(id).Header().IsEmpty()) {
+                cc->Outputs().Get(id).SetHeader(cc->Inputs().Get(id).Header());
+            }
+        }
+        if (cc->OutputSidePackets().NumEntries() != 0) {
+            for (CollectionItemId id = cc->InputSidePackets().BeginId();
+                 id < cc->InputSidePackets().EndId(); ++id) {
+                cc->OutputSidePackets().Get(id).Set(cc->InputSidePackets().Get(id));
+            }
+        }
+        cc->SetOffset(TimestampDiff(0));
+        MLOG("InputSidePacketUserTestCalc Open end");
+        return absl::OkStatus();
+    }
+
+    absl::Status Process(CalculatorContext* cc) final {
+        MLOG("InputSidePacketUserTestCalc process start");
+        if (cc->Inputs().NumEntries() == 0) {
+            return tool::StatusStop();
+        }
+        ov::Tensor* intTensor = new ov::Tensor(ov::element::Type_t::i64, {1});
+        cc->Outputs().Tag(INT64_TAG).Add(intTensor, cc->InputTimestamp());
+        *((int64_t*)intTensor->data()) = int64Param;
+        std::cout << intTensor->get_byte_size() << std::endl;
+        ov::Tensor* boolTensor = new ov::Tensor(ov::element::Type_t::boolean, {1});
+        *((bool*)boolTensor->data()) = boolParam;
+        std::cout << boolTensor->get_byte_size() << std::endl;
+        cc->Outputs().Tag(BOOL_TAG).Add(boolTensor, cc->InputTimestamp());
+        // there is no string to/from tensor in ovms
+        ov::Tensor* stringTensor = new ov::Tensor(ov::element::Type_t::i32, {1});
```

Comment:
You can create u8 tensor with shape {len} to return string.

Reply:
Yes I know but I decided that since it is not final support for string anyway I didn't want to bother with intricacies of OV::Tensor U8 -> string conversion.

Reply:
@atobiszei 
tensor conversion to string is one liner: `std::string my_str(tensor.data(), tensor.size());`